README.md

```diff
@@ -16,17 +16,17 @@ This is a command-line tool written in Java that performs the following steps:
 ### Run it
 To validate a GTFS dataset on your computer:
 
-`java -jar gtfs-validator-v2.0.0_cli.jar -i /myDirectory/gtfs.zip -o output -f ca-myFeedName` 
+`java -jar gtfs-validator-v2.0.0_cli.jar -i /myDirectory/gtfs.zip -o output -c ca` 
 
 To download and validate a GTFS dataset from a URL:
 
-`java -jar gtfs-validator-v2.0.0_cli.jar -u https://www.abc.com/gtfs.zip -o output -f ca-myFeedName`
+`java -jar gtfs-validator-v2.0.0_cli.jar -u https://www.abc.com/gtfs.zip -o output -c ca`
 
 where:
 * `--input` or `-i`: the path to the GTFS file (e.g., `/myDirectory/gtfs.zip`)
 * `--url` or `-u`: the fully qualified URL to the GTFS file (e.g., `https://www.abc.com/gtfs.zip`)
 * `--output` or `-o`: the path where the validation report will be stored (e.g., `output`)
-* `--feed_name` or `-f`: the name of the feed as a valid [ISO two letter country code](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2), followed by `-`, followed by a user-defined name for the feed. (e.g., `ca-myFeedName`, `us-myFeedName`)
```

Comment:
So what is the behavior of the code in this PR when someone passes in `-f` as a parameter?

We should provide some indication that `-f` is no longer supported so it the user doesn't continue to use `-f`, not realizing that it no longer does anything. I would consider returning false from `CliParametersAnalyzer.isValid()` if `-f` is provided, saying to use `-c` instead.

Reply:
> So what is the behavior of the code in this PR when someone passes in -f as a parameter?

As of now, an exception is raised if `-f` is provided. 
```
Exception in thread "main" com.beust.jcommander.ParameterException: Unknown option: -f
	at com.beust.jcommander.JCommander.parseValues(JCommander.java:742)
	at com.beust.jcommander.JCommander.parse(JCommander.java:282)
	at com.beust.jcommander.JCommander.parse(JCommander.java:265)
	at org.mobilitydata.gtfsvalidator.cli.Main.main(Main.java:49)
```

> I would consider returning false from CliParametersAnalyzer.isValid() if -f is provided, saying to use -c instead.

Done in https://github.com/MobilityData/gtfs-validator/pull/851/commits/251cb314de0d04521bc7f74af14fa18264fc1f90